Controllers/__init__.py

```diff
@@ -30,6 +30,7 @@ class template_controller(ABC):
     
     def __init__(
         self,
+        dt: float,
```

Comment:
Controller gets a "dt" argument. This is the control frequency/interval.

We need it because then, the controller can force this dt onto its subcomponents, specifically the predictor.

Before, the predictor had its own dt from config, but it should be consistent with the controller it is used in.

@marcinpaluch1994  Agree?